.storybook/main.js

```diff
@@ -0,0 +1,96 @@
+// // @ts-check
+const TsconfigPathsPlugin = require('tsconfig-paths-webpack-plugin');
+const path = require('path');
+
+module.exports = {
+  addons: [
+    '@storybook/addon-actions',
+    '@storybook/addon-links',
+    '@storybook/addon-a11y/register',
+    'storybook-addon-performance/register',
+    '@storybook/addon-knobs/register',
+  ],
+  webpackFinal: async (config, { configType }) => {
+    const tsPaths = new TsconfigPathsPlugin({
+      configFile: path.resolve(__dirname, '../tsconfig.base.json'),
+    });
+
+    const customRules = [
+      {
+        test: /\.(ts|tsx)$/,
+        use: [
+          {
+            loader: require.resolve('ts-loader'),
```

Comment:
You mentioned in proposal that we will use Babel for complication, do you plan to use `babel-loader` instead? 🐱

Reply:
Oops, it seems that I see the plan few lines below 💭